src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -211,7 +226,10 @@ class AddressValidationModal extends React.Component {
               this.renderAddressOption(address, String(index)),
             )}
           {this.renderPrimaryButton()}
-          <button className="usa-button-secondary" onClick={closeModal}>
+          <button
```

Comment:
NIT: Should we add `type="button"` here since [`submit` is the default if the attribute is not specified, or if the attribute is dynamically changed to an empty or invalid value.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type)